### PR TITLE
Fixed xref to Rules Development Guide

### PR DESCRIPTION
**Pull request by RichardHoch**
MTA 5.1.3 (current release)
Fixed a bad xref and corrected two minor typos.

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick: Please review

**Comment by apinnick** [comment]
Would you mind removing the ifndef from the story points module? An xref should be in the assemblies, not a module. Otherwise, it is very hard to keep track of them.

**Comment by RichardHoch** [comment]
> Would you mind removing the ifndef from the story points module? An xref should be in the assemblies, not a module. Otherwise, it is very hard to keep track of them.

Done; master.adocs for Maven and CLI guides modified; bccutil run on to check that the docs build  properly.

File: docs/maven-guide/master.adoc [hunk 1]
```diff
@@ -53,6 +53,8 @@ include::topics/maven-logging-properties.adoc[leveloffset=+2]
 // Rule Story Points
 include::topics/about-story-points.adoc[leveloffset=+2]
 
+For more information on categorizing tasks, see link:{ProductDocRulesGuideURL}/rule_categories_rules-development-guide#rule_categories_rules-development-guide[Using Custom Rule Categories].
```
Comment (apinnick) [inline docs/maven-guide/master.adoc:56]: Just noticed that the capitalization of the section title needs to be fixed.

File: docs/cli-guide/master.adoc [hunk 2]
```diff
@@ -60,6 +60,8 @@ include::topics/tech-tags.adoc[leveloffset=+2]
 // Rule Story Points
 include::topics/about-story-points.adoc[leveloffset=+2]
 
+For more information on categorizing tasks, see link:{ProductDocRulesGuideURL}/rule_categories_rules-development-guide#rule_categories_rules-development-guide[Using Custom Rule Categories].
```
Comment (apinnick) [inline docs/cli-guide/master.adoc:63]: Capitalization of section title: Title case > Sentence case

**Comment by RichardHoch** [comment]
@apinnick: Capitalization fixed.

**Comment by apinnick** [comment]
LGTM. Please squash your commits and I will merge.

**Comment by RichardHoch** [comment]
@apinnick: Squashed.

**Pull request merged**